posthog/warehouse/models/external_data_schema.py

```diff
@@ -309,6 +309,45 @@ def filter_postgres_incremental_fields(columns: list[tuple[str, str]]) -> list[t
     return results
 
 
+def get_postgres_row_count(
```

Comment:
This is all fine to live here right now, but we need to figure out a more longer-term solution for these SQL sources - they're all kinda over the place right now - some future pipeline work

Reply:
Yeah, I noticed it's a little chaotic with all of these definitions. Certainly worth consolidating some base interfaces for the corresponding providers.